app/src/main/kotlin/com/wire/android/ui/markdown/MarkdownComposer.kt

```diff
@@ -158,9 +159,8 @@ fun inlineChildren(
                 annotatedString.pop()
             }
 
-            is HardLineBreak -> {
+            is HardLineBreak, is SoftLineBreak -> {
```

Comment:
That is a blessing from commonmark. I wasn't expecting it to be so simple!

Nice find.